modules/monitoring-creating-cluster-id-labels-for-metrics.adoc

```diff
@@ -0,0 +1,95 @@
+// Module included in the following assemblies:
+//
+// * monitoring/configuring-the-monitoring-stack.adoc
+
+:_content-type: PROCEDURE
+[id="creating-cluster-id-labels-for-metrics_{context}"]
+= Creating cluster ID labels for metrics
+
+You can configure cluster ID labels for metrics for default platform monitoring and for user workload monitoring.
+
+For default platform monitoring, you add cluster ID labels for metrics in the `write_relabel` settings for remote write storage in the `cluster-monitoring-config` config map in the `openshift-monitoring` namespace. 
```

Comment:
Minimalism: do you need either of these paragraphs for default platform monitoring and user workload monitoring? You very well might. 

These actions aren't included in the procedure nor in the prereqs. They don't seem necessary for the intro/abstract which is meant to set the procedure module up. 

Maybe this content would be better in a `Verification` section? So that way, users have completed the procedure on creating cluster ID labels and can verify that they can add these cluster ID labels to their monitoring workloads (default platform or user workload)? 

What do you think: keep where it is, omit it, move it to a verification section?

Reply:
Makes sense, Brian. In that case, leave as is :)